circle.yml

```diff
@@ -7,6 +7,8 @@ machine:
 dependencies:
   bundler:
     without: [oracle]
+  pre:
+    - cp config/secrets.yml.template config/secrets.yml
```

Comment:
Confused what this is for--I thought the usual way to set secrets for circle was through their UI.